.buildkite/pipeline.yml

```diff
@@ -0,0 +1,33 @@
+steps:
+  - command: .buildkite/scripts/run_autoformat.py
+  - wait
+  - command: .buildkite/scripts/run_job.py internal_model
+  - command: .buildkite/scripts/run_job.py display
+  - command: .buildkite/scripts/run_job.py elasticsearch
+  - command: .buildkite/scripts/run_job.py big_messaging
+  - command: .buildkite/scripts/run_job.py ingestor_common
+  - command: .buildkite/scripts/run_job.py transformer_common
+  - wait
+  - command: .buildkite/scripts/run_job.py api
+  - command: .buildkite/scripts/run_job.py reindex_worker
+  - command: .buildkite/scripts/run_job.py ingestor_works
+  - command: .buildkite/scripts/run_job.py ingestor_images
+  - command: .buildkite/scripts/run_job.py id_minter
+  - command: .buildkite/scripts/run_job.py matcher
+  - command: .buildkite/scripts/run_job.py merger
+  - command: .buildkite/scripts/run_job.py recorder
+  - command: .buildkite/scripts/run_job.py transformer_miro
+  - command: .buildkite/scripts/run_job.py transformer_sierra
+  - command: .buildkite/scripts/run_job.py transformer_mets
+  - command: .buildkite/scripts/run_job.py transformer_calm
+  - command: .buildkite/scripts/run_job.py sierra_reader
+  - command: .buildkite/scripts/run_job.py sierra_bib_merger
+  - command: .buildkite/scripts/run_job.py sierra_item_merger
+  - command: .buildkite/scripts/run_job.py sierra_items_to_dynamo
+  - command: .buildkite/scripts/run_job.py mets_adapter
+  - command: .buildkite/scripts/run_job.py calm_adapter
+  - command: .buildkite/scripts/run_job.py inference_manager
+  - command: .buildkite/scripts/run_job.py feature_inferrer --changes-in pipeline/inferrer/feature_inferrer
+  - command: .buildkite/scripts/run_job.py snapshot_generator
+  - wait
+  - command: .buildkite/scripts/complete_build.py
```

Comment:
Maybe I am being silly, but if we have two branches, branch `A` and `B`. If `B` finishes, then `A` finishes, would `B` not be latest although maybe not latest?

Perhaps I am missing the point as to why it exists though.

Reply:
Yes you are correct. A race condition might occur producing an inaccurate value which could result in more tests/builds being run than necessary (but not the other way round I think). I'm ok with that outcome for now.

Reply:
Super, just checking I understood.